### ISSUE TITLE
Binance Desktop App Native ARM support

### ISSUE THREAD

**Issue by ishariqriaz** [binance/desktop#25]
Hello its been 2 years since Apple updated their hardware to ARM archetecture.. Electron announced their support 2 years ago as well.. but binance app development team has failed to recognize this .. can you guys please update the desktop app for mac OS ? the app does work right now but its battery and power hungry due to which both battery and performance are affected..

**Comment by marcelek** [comment]
Hi guys,
Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.

I have always seen Binance as a leader in technology solutions for crypto - what's happening?

**Comment by panghu-huang** [comment]
> Hi guys, Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.
> 
> I have always seen Binance as a leader in technology solutions for crypto - what's happening?

https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg

**Comment by ishariqriaz** [comment]
> > Hi guys, Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.
> > I have always seen Binance as a leader in technology solutions for crypto - what's happening?
> 
> https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg

thanks but still not available in github or their website :O perhaps still in beta?

**Comment by panghu-huang** [comment]
> > > Hi guys, Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.
> > > I have always seen Binance as a leader in technology solutions for crypto - what's happening?
> > 
> > 
> > https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg
> 
> thanks but still not available in github or their website :O perhaps still in beta?

Yes, we forgot to upload it to Github and this version is only available to users in grayscale

**Comment by ishariqriaz** [comment]
> > > > Hi guys, Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.
> > > > I have always seen Binance as a leader in technology solutions for crypto - what's happening?
> > > 
> > > 
> > > https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg
> > 
> > 
> > thanks but still not available in github or their website :O perhaps still in beta?
> 
> Yes, we forgot to upload it to Github and this version is only available to users in grayscale

i installed it , switch account is not working + it seems to be only using CPU not GPU .. pls look into it thanks

**Comment by ishariqriaz** [comment]
> > > > Hi guys, Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.
> > > > I have always seen Binance as a leader in technology solutions for crypto - what's happening?
> > > 
> > > 
> > > https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg
> > 
> > 
> > thanks but still not available in github or their website :O perhaps still in beta?
> 
> Yes, we forgot to upload it to Github and this version is only available to users in grayscale

any update?

**Comment by panghu-huang** [comment]
> https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg

switch account failed has been fixed on https://download.binance.com/electron-desktop/mac/production/binance-1.40.0-arm64.dmg

**Comment by panghu-huang** [comment]
> > > > > Hi guys, Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.
> > > > > I have always seen Binance as a leader in technology solutions for crypto - what's happening?
> > > > 
> > > > 
> > > > https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg
> > > 
> > > 
> > > thanks but still not available in github or their website :O perhaps still in beta?
> > 
> > 
> > Yes, we forgot to upload it to Github and this version is only available to users in grayscale
> 
> any update?

switch account failed has been fixed on https://download.binance.com/electron-desktop/mac/production/binance-1.40.0-arm64.dmg

**Comment by ishariqriaz** [comment]
> > > > > > Hi guys, Can you make an Apple Silicon version? Do you need help? I think I can help you with Electron m1-arm version.
> > > > > > I have always seen Binance as a leader in technology solutions for crypto - what's happening?
> > > > > 
> > > > > 
> > > > > https://download.binance.com/electron-desktop/mac/production/binance-1.39.0-arm64.dmg
> > > > 
> > > > 
> > > > thanks but still not available in github or their website :O perhaps still in beta?
> > > 
> > > 
> > > Yes, we forgot to upload it to Github and this version is only available to users in grayscale
> > 
> > 
> > any update?
> 
> switch account failed has been fixed on https://download.binance.com/electron-desktop/mac/production/binance-1.40.0-arm64.dmg

thanks man still not posted on website or github :(

**Comment by mbunal** [comment]
I wonder, m1 App run full screen on iPad Pro?

**Comment by ishariqriaz** [comment]
> I wonder, m1 App run full screen on iPad Pro?

lol obv no its not built that way its made for desktop environment

**Comment by mbunal** [comment]
Lol u too. Apple is about to merge desktop and tablet os. So why not?

**Comment by ishariqriaz** [comment]
> 

theres a difference between wanting to and already happened, u know they cant release anything unless apple does it first right? its just not possible with apple's ecosystem until apple allows it.. best they can do is release iPad app which would be like iPhone app just scaled properly
